content/engine/install/sles.md

```diff
@@ -24,17 +24,19 @@ aliases:
 download-url-base: https://download.docker.com/linux/sles
 ---
 
+> **Note**
```

Comment:
[nit] Should this be 
```suggestion
> **Warning**
```
since it's using the warning styling?

Reply:
No, this isn't a warning, just a very loud note :-) Warnings are for when something bad can happen. This is just trying to prevent people from wasting their time